### PR TITLE
fix: Invalid time zone specified: Etc/Unknown

### PR DESCRIPTION
**Pull request by anikdhabal**
## What does this PR do?

- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [x] N/A-I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A-I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] N/A-I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: packages/features/bookings/lib/timePreferences.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ type TimePreferencesStore = {
 };
 
 const timezoneLocalStorageKey = "timeOption.preferredTimeZone";
+const currentTimezone = dayjs.tz.guess() !== "Etc/Unknown" ? dayjs.tz.guess() : "Europe/London";
```
Comment (anikdhabal) [inline packages/features/bookings/lib/timePreferences.ts:15]: `Etc/Unknown` timezone states that timzone information cannot be determined based on user's preferences.
More info about the issue:- https://github.com/iamkun/dayjs/issues/2560

Comment (anikdhabal) [reply packages/features/bookings/lib/timePreferences.ts:15]: Yeah, i will update it

**Review by emrysal — DISMISSED** [review]
Requesting changes until the feedback by @Udit-takkar is addressed

File: packages/features/bookings/lib/timePreferences.ts (hunk 1 above)
Comment (anikdhabal) [reply packages/features/bookings/lib/timePreferences.ts:15]: done

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**